.env.example

```diff
@@ -27,3 +27,8 @@ DB_HOST=127.0.0.1
 # If your owid-content repos is not in the same folder as etl, define the path here.
 # NOTE: The path should not point to the root, but to the explorers folder, where the tsv files are.
 # EXPLORERS_DIR="/full_path_to_owid_content_folder/explorers/"
+
+# Metabase credentials.
+# Uncomment and fill out the following variables if you want to interact with the Metabase API to gather analytics.
+# METABASE_API_KEY=***
+# METABASE_URL=***
```

Comment:
how can one get these? Also, will staging servers have this set by default? Otherwise some of the tools wouldn't work, right?

Reply:
Yes, good point, staging servers won't have those credentials, I'll look into it.